packages/ui-extensions/src/surfaces/checkout/api/address-autocomplete/shared.ts

```diff
@@ -39,7 +39,7 @@ export interface AddressAutocompleteSuggestion {
   matchedSubstrings?: MatchedSubstring[];
 
   /**
-   * The formatted address that will auto-fill the remaining address fields.
+   * The address object used to auto-populate the remaining address fields.
```

Comment:
Seems pretty inoffensive, but can we keep this change as part of the release to `unstable` @hheyhhay ?

Original PR: https://github.com/Shopify/checkout-web/pull/34241